decentraland/unity-renderer
# feat: channel searching window

## What does this PR change?

Includes the implementation of a new channel search window. It is accessible by the search channels button in the conversation list.

https://user-images.githubusercontent.com/56365551/178600467-7915894c-079d-4819-ba7a-c7dec6fd4b64.mp4

## How to test the changes?

1. Open the conversation list
2. Click on search channels button
3. 30 channels should be loaded first
4. Scroll down to load more
5. Use the search bar
6. Clear the search bar
7. Go back or close the window

## Our Code Review Standards

https://github.com/decentraland/unity-renderer/blob/master/docs/code-review-standards.md
